lib/service_check.pm

```diff
@@ -1,3 +1,9 @@
+=head1 service_check
+
+check service status before and after migration
```

Comment:
check service status *or service function* before and after migration, since we not only check status ;)

Reply:
updated, thanks for your feedback!